src/familiar/itemFamiliar.ts

```diff
@@ -12,7 +12,12 @@ export function bestFairy(): Familiar {
   if (!bestNonCheerleaderFairy) {
     const viableFairies = Familiar.all()
       .filter(
-        (f) => have(f) && findFairyMultiplier(f) && f !== $familiar`Steam-Powered Cheerleader`
+        (f) =>
+          have(f) &&
+          findFairyMultiplier(f) &&
```

Comment:
i took a brief look at libram and i expect this works properly but you know how i am in life and must have known i would ask this -- what does findFairyMultiplier do regarding special-case drop fams, like the peppermint rhino that has item drop & extra +candy drop?